tsp/tsp_client.py

```diff
@@ -252,6 +253,52 @@ def fetch_timegraph_tree(self, exp_uuid, output_id, parameters=None):
             print("failed to get tree: {0}".format(response.status_code))
             return TspClientResponse(None, response.status_code, response.text)
 
+    def fetch_xy_tree(self, exp_uuid, output_id, parameters=None):
+        '''
+        Fetch XY tree, Model extends Entry
+        :param exp_uuid: Experiment UUID
+        :param output_id: Output ID
+        :param parameters: Query object
+        :returns: :class:  `TspClientResponse <GenericResponse>` object XY entry response with entries and headers
+        :rtype: TspClientResponse
+        '''
+        api_url = '{0}experiments/{1}/outputs/XY/{2}/tree'.format(
+            self.base_url, exp_uuid, output_id)
+
+        params = parameters
+        if (parameters is None):
+            requested_times = [0, 1]
+            my_parameters = {self.REQUESTED_TIME_KEY: requested_times}
+            params = {self.PARAMETERS_KEY: my_parameters}
```

Comment:
when the parameters is None we should pass an empty params instead, line 272, params = { }

This requires Trace Server patches to be merged to get a successful reply:
- https://bugs.eclipse.org/bugs/show_bug.cgi?id=575950
- https://bugs.eclipse.org/bugs/show_bug.cgi?id=575978

Reply:
-All done by now. I just force-pushed a rebase of this branch on latest master, along (only) with the new 6b9db2c on top that applies the above comment.